src/app/tests/Extensions.ts

```diff
@@ -81,6 +82,60 @@ expect.extend({
         };
     },
 
+    toBeOk(received: unknown) {
+        if (typeof received !== "object" || !received) {
```

Comment:
This is just not worth the effort lol, it's not like we're publishing this library, just do
```ts
if (!received || typeof received !== "object" || !("ok" in received)) {
  return {
    message: () => "supplied value is not a Result",
    pass: false,
  };
}
const result = received as Result;
...
```

Reply:
Well I did already put in the effort though but I can simplify it